vspace/src/main/java/edu/asu/diging/vspace/core/services/impl/ImageService.java

```diff
@@ -116,8 +127,15 @@ public ImageData getImageDimensions(IVSImage image, int width, int height) {
         else {
             sortingParameters = sortingParameters.descending();
         }
+        return sortingParameters;
+    }
+
+    @Override
+    public List<IVSImage> getImages(int pageNo, ImageCategory category, String sortedBy, String order) {
```

Comment:
is this method any different from the above `getImages`? it's only the find line, isn't it? Should be one method.